apps/sms/js/thread_ui.js

```diff
@@ -1497,6 +1492,7 @@ var ThreadUI = {
       }
       taskQueue.push(() => {
         if (!this._stopRenderingNextStep) {
+          messageMap.set(message.id, message);
```

Comment:
shouldn't we simply use the existing Threads.registerMessage ?

BTW in `Threads.registerMessage` we could move the first line (`Thread.create(message)`) to the if block.